samples/android/GleanCrashExample/app/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.26"
+}
+
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    defaultConfig {
+        applicationId "org.mozilla.gleancrashexample"
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'androidx.appcompat:appcompat:1.0.2'
```

Comment:
```suggestion
    implementation "androidx.appcompat:appcompat:$rootProject.versions.androidx_appcompat"
```

Reply:
Please use the the same version we're using in this repo for the other dependencies, as I did with this one :)

Reply:
This is a standalone example, if I make this so that it relies upon `$rootProject...` then it can no longer be copied out and used without the entire repo.

Reply:
Unfortunately, since this is living in the mozilla/glean repo, to make sure this is always up to date and not crashing, we need to sync up our dependencies.

It's a bit weird that a sample, that is listed out when importing the whole repo, requires a different set of dependencies. That would be ok, I guess, if it lived outside of this repo.

Reply:
This will no longer be valid after I remove the example application in favor of more examples inline with the documentation.